### ISSUE TITLE
Windows installer 4.1.0 misses ICU dlls

### ISSUE THREAD

**Issue by db4** [UB-Mannheim/tesseract#14]
I've  installed
https://digi.bib.uni-mannheim.de/tesseract/tesseract-ocr-w64-setup-v4.1.0.20190314.exe
When I try to run `unicharset_extractor.exe` or `combine_lang_model.exe`  they fail with

![image](https://user-images.githubusercontent.com/6112699/60386681-1d177700-9aa1-11e9-9772-a0b6c7189f5e.png)
![image](https://user-images.githubusercontent.com/6112699/60386688-2c96c000-9aa1-11e9-943e-bf639a6d08c7.png)

Other EXEs from `C:\Program Files\Tesseract-OCR` start normally.

**Comment by stweil** [comment]
Yes. But why do you want to use an incomplete old version? Chose one of the current installers: https://github.com/UB-Mannheim/tesseract/wiki.

**Comment by db4** [comment]
Why is it old? It's the last general-purpose 4.1.x version build. More recent versions are 5.0.0 alpha (no, I need something more stable) and two special "branded" editions (built only for "Bibliotheken verändern" and "ELAG 2019", right?)

**Comment by stweil** [comment]
"Branded" just means that the installer shows the logo of those conferences. Otherwise that are fixed newer 4.1 versions. And 5.0 is nearly identical to 4.1 – it only includes some API incompatibilities like removed code. It is as stable as 4.1.

**Comment by db4** [comment]
OK, thanks for the clarification.  I've tried https://digi.bib.uni-mannheim.de/tesseract/tesseract-ocr-w64-setup-v4.1.0-elag2019.exe and although it contains ICU DLLs `unicharset_extractor.exe` still does not work:
![image](https://user-images.githubusercontent.com/6112699/60392224-1de2f400-9b07-11e9-820a-c432da0e3c38.png)

**Comment by stweil** [comment]
Thank you for testing. I did not notice that because it works with Wine on Linux.

I updated the [Wiki](https://github.com/UB-Mannheim/tesseract/wiki) now and still suggest to use the latest installer which contains the best (and also fastest) available Tesseract binaries for Windows. The latest version also gets the best support: if you find and report a problem, we'll try to fix it and make a new installer.
